app/code/Magento/SalesRule/Api/Data/RuleInterface.php

```diff
@@ -59,14 +61,14 @@ public function setName($name);
     /**
      * Get display label
      *
-     * @return \Magento\SalesRule\Api\Data\RuleLabelInterface[]|null
```

Comment:
@Bartlomiejsz Please return back fully qualified return types and param definition in doc block. For future, please read [this](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/service-contracts/service-to-web-service.html#service-interface-requirements)

Reply:
@Stepa4man clear. done now